org.eclipse.winery.topologymodeler.ui/src/app/canvas/canvas.component.ts

```diff
@@ -1746,8 +1741,8 @@ export class CanvasComponent implements OnInit, OnDestroy, AfterViewInit, DoChec
      */
     private handleRelSideBar(conn: any, newRelationship: TRelationshipTemplate): void {
```

Comment:
Why is conn still `any`?

Reply:
conn is an connection object generated by jsPlumb for its inner management of UI elements